### PR TITLE
fix gallery

### PR DESCRIPTION
**Pull request by deargle**
@sauln can you add me (deargle) to https://readthedocs.org/projects/kepler-mapper/ ? I have the gallery fixed locally, want to make sure it works live before merging.

### CODE REVIEW

**Comment by sauln** [comment]
@deargle  you should have access now.

**Comment by deargle** [comment]
Hurray! See https://kepler-mapper.scikit-tda.org/en/fix-gallery/examples.html , gallery is fixed

**Comment by deargle** [comment]
@sauln I made the thumbnails on the gallery page (1) d3 vis instead of the default matplotlib capture, and (2) bigger, but still small enough for there to be two per row. In favor?

**Comment by deargle** [comment]
Closes #218

File: examples/plot_digits.py [hunk 1]
```diff
@@ -60,7 +64,7 @@
 # Create the visualizations (increased the graph_gravity for a tighter graph-look.)
 print("Output graph examples to html" )
 # Tooltips with image data for every cluster member
-mapper.visualize(graph,
+vis_1 = mapper.visualize(graph,
```
Comment (sauln) [inline examples/plot_digits.py:67]: You mind running `black` on these examples? Looks like the formatting has drifted.

**Review by sauln — APPROVED** [review]
Looks great! Thank you 🙇 

One small comment on formatting, a quick run of black over the examples would be great I think.

File: examples/plot_digits.py (hunk 1 above)
Comment (deargle) [reply examples/plot_digits.py]: Done, and will do for future changes too

**Pull request merged**